operations/network/customer_accessible_networks/can_to_chn/network/chn_enable.md

```diff
@@ -473,6 +483,16 @@ Administrators should enable CFS for UAN, ensure plays run successfully and then
    done
    ```
 
+    Alternatively, if running CSM 1.6.2 or higher, enable multiple components at once by using the `updatemany` command. Refer to
+    [Managing many components](../../../../configuration_management/CFS_Commands_Cheat_Sheet.md#managing-many-components)
+    for complete `updatemany` command requirements and usage.
+
+   ```bash
```

Comment:
The text paragraph is indented 1 space further than the code blocks above and below.

````suggestion
   Alternatively, if running CSM 1.6.2 or higher, enable multiple components at once by using the `updatemany` command. Refer to
   [Managing many components](../../../../configuration_management/CFS_Commands_Cheat_Sheet.md#managing-many-components)
   for complete `updatemany` command requirements and usage.

   ```bash
````